docs/dev_guide/pr_review_procedure.rst

```diff
@@ -26,6 +26,30 @@ Before the ‘merge’ button is clicked the following criteria must be met:
 
 It is important that approval for merging the PR is done on the comment thread, as this becomes part of the ‘permanent record’, this includes in during community meetings or in chat.
 
+Continuous Integration
+======================
+
+Currently we have a variety of bots or services that respond or activate on an opened pull request.
+While we try not to change them, they have undergone several changes with the aim of making them clear and focused on specific issues.
+
+*  pep8speaks: Performs a PEP8 check on any submitted code.
+*  CircleCi: Tests to see if sunpy installs and builds the documentation.
+*  Giles: Returns a link if the documentation does build successfully.
+*  Travis: Runs our test suite to make sure it passes on Linux and mac OS.
+*  AppVeyor: Runs our test suite to make sure it passes on Windows.
+*  CodeCov: Checks how many lines of the code lack test coverage.
+
+We support several custom tags you can add anywhere in the commit message.
+Please use these tags extensively, especially for documentation PRs and WIP commits.
+
+*  [skip ci] or [ci skip]  - Skips Appveyor and Travis.
+*  [skip appveyor] - Skips Appveyor only.
+*  [skip travis] or [travis skip] -  These terminate the Travis builds before the install steps, effectively skipping testing.
+*  [docs only] or [build docs] - These terminate all non documentation build jobs in Travis before the install steps.
```

Comment:
We aren't doing these in Circle now?

Reply:
Well it also stops app and travis from running the tests.